mobile/src/TableCell.js

```diff
@@ -12,11 +12,17 @@ export default class TableCell extends Component {
   }
 
   renderCell = item => {
-    const { primaryColor, currentUser } = this.props
+    const { primaryColor, currentUser, answerButtonPrompt } = this.props
     const voteCount = item.questionId
       ? this.props.votesByAnswer[item.id] || 0
       : this.props.votesByQuestion[item.id] || 0
 
+    let buttonPrompt = this.props.commentsTotal === 1
+      ? t('one_answer')
+      : t('answers_count', { count: this.props.commentsTotal })
+    if (answerButtonPrompt) buttonPrompt = `${this.props.commentsTotal} ${answerButtonPrompt}${this.props.commmentsTotal === 1 ? "": "s"}`
+    console.log(this.props.commentsTotal)
```

Comment:
please review every commit before committing and review every PR before submitting (as if you were doing a code review for someone else)